x-pack/plugins/ml/public/file_datavisualizer/components/fields_stats/fields_stats.js

```diff
@@ -46,38 +46,53 @@ function createFields(results) {
   const {
     mappings,
     field_stats: fieldStats,
+    column_names: columnNames,
     num_messages_analyzed: numMessagesAnalyzed,
     timestamp_field: timestampField,
   } = results;
 
   let fields = [];
```

Comment:
I think in this case you could get rid of using `let` by just `return`ing `fields` in the places where they're final, see also https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#returnthrow-early-from-functions